crates/noirc_evaluator/src/ssa_refactor/ir/function.rs

```diff
@@ -15,7 +15,7 @@ use noirc_errors::Location;
 /// To reference external functions, one must first import the function signature
 /// into the current function's context.
 #[derive(Debug)]
-pub(crate) struct Function {
+pub struct Function {
```

Comment:
Ah, these publicity changes were remnants of when I added a `println!("{ssa_functions}")` to the driver. The publicity can stay since they need to be public to access them anyway.